src/main/java/com/ticketservice/annotation/NullableWarningAnnotation.java

```diff
@@ -0,0 +1,15 @@
+package com.ticketservice.annotation;
+
+import java.lang.reflect.Field;
+
+public class NullableWarningAnnotation {
+
```

Comment:
You don't really need separated class for this, checkFieldsForNull() could be located parent within Entity class, and then you just should call it in children's constructor eg.  in Ticket class